src/ml_board/frontend/Dashboard/src/components/dashboard/table/Table.tsx

```diff
@@ -53,6 +57,7 @@ export default function Table({ colNames, rows }: { colNames: string[], rows: an
         animateRows={true} // Optional - set to 'true' to have rows animate when sorted
         rowSelection="multiple"
         onCellClicked={onCellClicked}
+        // getRowStyle={getRowStyle}
```

Comment:
legacy code

Reply:
intentionally left to expose what we can do in the future if ever needed! (like lines 56, 52, 50, ... etc)